### DISCUSSION TITLE
Is sqlite support staying long term?

### DISCUSSION THREAD

**Discussion by supertin** [rspamd/rspamd#3593] [General]
My server is very low volume, and I've been using sqlite since the start. Simple (from my perspective as a user) and works fine for our low volume.

So my question: Is there any plans to remove sqlite support? Or will it remain in the long term for low volume users like me?

**Comment by vstakhov** [comment]
There are no current plans to drop sqlite3 support. I agree that it is one of the best possible options for low memory systems (e.g. embedded systems). Rspamd is not particularly designed for such systems but it does not mean that this support should be dropped.

**Comment by supertin** [comment]
Well that's excellent news for me :laughing:

Fantastic program, BTW.
